connector/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@1hive/connect-conviction-voting",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Hey, I think this should be a major bump. The API broke: `config` -> `onConfig` and it's not a backward compatible change.

Reply:
so you think about doing `0.3.0`?